apps/server/src/shared/testing/factory/tldraw.ws.factory.ts

```diff
@@ -3,10 +3,14 @@ import WebSocket from 'ws';
 
 export class TldrawWsFactory {
 	public static createWsSharedDocDo(): WsSharedDocDo {
-		return { conns: new Map(), destroy: () => {} } as WsSharedDocDo;
+		return { connections: new Map(), destroy: () => {} } as WsSharedDocDo;
 	}
 
 	public static createWebsocket(readyState: number): WebSocket {
```

Comment:
This is in testing therefore the cast maybe also the double cast is nearly "ok", but in the end we have a high delta between the real type and what do you use. 
I think in general if you only need a subset then a adapter / own interface that define only what we need is the better way. 

For the readyState a enum should be use. If i pass 5, or -1 then it make no sense for value.